### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
Global visibility card to broadcast event, and render json data

### CODE REVIEW

File: assets/css/globalVisibilityCard.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  
+  width: 225px;
+  left: 745px;
+  top: 561px;
+  border: 1px solid #4EDCE6;
+  box-shadow: 0px 4px 4px rgba(24, 192, 203, 0.25);
+  
+  }
+  
+  .random{
```
Comment (cs-pranali-chavan) [inline assets/css/globalVisibilityCard.css]: remove unused class

**Review by cs-pranali-chavan — APPROVED** [review]

**Review by cs-pranali-chavan — CHANGES_REQUESTED** [review]
Looks good

File: assets/css/globalVisibilityCard.css (hunk 1 above)
Comment (cs-yash-patodkar) [reply assets/css/globalVisibilityCard.css]: Done

**Review by cs-pranali-chavan — APPROVED** [review]
Changes look good

**Pull request closed without merging**